components/AntHiveBotHorizontal.vue

```diff
@@ -88,7 +88,7 @@
             }}</span>
           </div>
           <div class="statistic">
-            <span>{{ $t("game.rt") }}:</span>
```

Comment:
This is incorrect it should be RT - response time (not average due to it's a response time for each tick )